src/CurrentBookRainbow/CurrentBookRainbow.js

```diff
@@ -34,12 +44,13 @@ const CurrentBookRainbow = ({data}) => {
             <path d="M735.732 14C728.986 16.9474 722.663 22.951 717.245 27.9225C711.135 33.5284 703.452 40.5738 699.277 48.0676C695.378 55.0671 691.647 62.7344 690.726 71.0815C690.222 75.6503 689.658 80.183 689.324 84.7809C688.811 91.8425 689.71 98.9289 688.051 105.873C686.722 111.434 685.877 118.02 683.573 123.142C679.652 131.86 673.996 140.194 667.04 146.248C661.307 151.238 654.05 154.061 647.474 157.512C640.324 161.265 633.115 164.888 626.58 169.749C614.434 178.785 607.151 195.367 600.499 209.08C596.877 216.547 595.543 226.7 594.735 235.056C593.908 243.603 593.049 251.986 592.883 260.654C592.577 276.643 594.874 296.645 583.857 308.773C579.87 313.162 576.296 317.891 572.184 322.162C567.806 326.709 562.49 330.361 557.724 334.463C550.729 340.483 542.314 344.567 534.84 349.829C524.77 356.917 515.973 367.828 510.864 379.572C504.938 393.191 503.257 407.73 503.936 422.872C504.579 437.193 511.233 450.798 511.066 465.19C510.86 482.928 504.218 501.021 495.117 515.329C486.406 529.024 472.777 537.615 461.335 548.386C457.612 551.891 453.384 554.504 450.546 558.967C447.433 563.86 444.665 570.141 442.755 575.772" stroke="#057196" stroke-width="22" stroke-linecap="round" stroke-linejoin="round"/>
             <path d="M580.911 14C574.066 16.7085 567.536 22.4861 561.946 27.2641C555.643 32.6518 547.717 39.4227 543.282 46.7652C539.139 53.6232 535.141 61.1547 533.927 69.4642C533.263 74.0125 532.54 78.5226 532.044 83.1059C531.283 90.1452 531.933 97.2587 530.031 104.141C528.507 109.651 527.431 116.203 524.949 121.241C520.723 129.816 514.778 137.946 507.613 143.752C501.708 148.537 494.357 151.103 487.663 154.321C480.386 157.82 473.054 161.187 466.352 165.816C453.896 174.42 446.035 190.736 438.905 204.206C435.023 211.541 433.332 221.641 432.231 229.964C431.105 238.476 429.952 246.824 429.481 255.48C428.613 271.449 430.205 291.519 418.769 303.252C414.63 307.499 410.892 312.099 406.632 316.222C402.097 320.613 396.656 324.076 391.749 328.008C384.547 333.779 375.994 337.564 368.339 342.56C358.026 349.29 348.851 359.885 343.332 371.442C336.931 384.844 334.74 399.315 334.887 414.471C335.026 428.806 341.197 442.637 340.525 457.014C339.696 474.734 332.422 492.582 322.824 506.561C313.636 519.942 299.714 528.048 287.9 538.41C284.056 541.782 279.739 544.246 276.746 548.605C273.463 553.387 270.476 559.566 268.369 565.127" stroke="#EB6825" stroke-width="20" stroke-linecap="round" stroke-linejoin="round"/>
+
```

Comment:
This file looks great! D3 is intense but we can do so much with it. Thanks for tinkering around with the x and y axis until it aligned with our svg!